services/postprocessing/pkg/config/defaults/defaultconfig.go

```diff
@@ -35,9 +35,10 @@ func DefaultConfig() *config.Config {
 			MaxRetries:           14,
 		},
 		Store: config.Store{
-			Store:    "memory",
+			Store:    "nats-js-kv",
+			Nodes:    []string{"127.0.0.1:9233"},
 			Database: "postprocessing",
-			Table:    "postprocessing",
+			Table:    "",
```

Comment:
I don''t really like an empty table, but since it is only used as a prefix it is ok here.

Reply:
Yes I was thinking the same. :+1: Table makes not much sense when using nats kv.